src/main/scala/is/hail/expr/Type.scala

```diff
@@ -606,6 +606,8 @@ case object TCall extends Type {
   override def desc: String = "A ``Call`` is a Hail data type representing a genotype call (ex: 0/0) in the Variant Dataset."
 
   override def scalaClassTag: ClassTag[java.lang.Integer] = classTag[java.lang.Integer]
+
+  override def str(a: Annotation): String = if (a == null) "." else Call.toString(a.asInstanceOf[Call])
```

Comment:
Also, call.toString already handles null.  Do we want ./. or .?  The former seems more traditional in VCF.

In the long term, we should literals in the expr lang for all types and str should generate the corresponding literal, but that's for another PR.